sys-process/rtirq/rtirq-20180209.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit systemd
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
```

Comment:
Please keep the ordering and grouping to the template:

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
```

i.e. move the empty line one variable lower.

Reply:
Fixed in 46bf5ad584d30d440f1b05969eee5bfc8589c35f.